Assets/_BForBoss/_Entities/Animation/Fighting Idle.fbx.meta

```diff
@@ -0,0 +1,768 @@
+fileFormatVersion: 2
```

Comment:
Do we actually need the whole fbx? or just the animation inside?

Reply:
They're skinless fbx's so I don't know that it matters? I can try to extract the anim file but I'm a little scared that will break things

Reply:
Nvm, it doesn't break things